JSDemos/Demos/TreeView/HierarchicalDataStructure/description.md

```diff
@@ -1 +1,5 @@
-The TreeView component displays a&nbsp;hierarchical collection of&nbsp;items using a&nbsp;traditional tree structure. TreeView nodes can be&nbsp;expanded and collapsed to&nbsp;display sub-items.
\ No newline at end of file
+To bind the TreeView to a hierarchical array, assign this array to the [items[]](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/items/) property. If you need to use a [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object and its diverse API, specify the [dataSource](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#dataSource) property instead.
```

Comment:
Instead of "diverse API", I would suggest being more specific:

```suggestion
To bind the TreeView to a local hierarchical array, assign this array to the [items[]](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/items/) property as shown in this demo. If you want to process the local array or bind the TreeView to remote data, you need to use the [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object and specify the [dataSource](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#dataSource) property instead of **items[]**.
```